packages/cli/src/controllers/auth.controller.ts

```diff
@@ -95,7 +101,22 @@ export class AuthController {
 		} else {
 			user = await handleEmailLogin(email, password);
 		}
+
 		if (user) {
+			if (user.mfaEnabled) {
+				const { decryptedRecoveryCodes, decryptedSecret } =
+					await this.mfaService.getRawSecretAndRecoveryCodes(user.id);
+
+				user.mfaSecret = decryptedSecret;
+				user.mfaRecoveryCodes = decryptedRecoveryCodes;
+
+				const isMFATokenValid =
+					(await this.validateMfaToken(user, mfaToken)) ||
+					(await this.validateMfaRecoveryCode(user, mfaRecoveryCode));
```

Comment:
That change would look better but would not work. 

1. The first condition would stop users for ever login in in they have MFA enable, because with MFA you do not login with both, token and recovery code, but with only one of then. So this will always throw error.
2. Leaving the first condition aside, checking individually like this would allow users to login event if the mfaToken or mfaRecovery code are invalid. And to account for that would make the code less readable the it currently is, because then I have throw 3 times instead of 1.